scripts/kfws

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
What's this script for?

Why not just tell users to create the YAMl directly and do kubectl apply?

If we have a script that creates and applies a resource, then we hide the declarative management.
With kfctl.sh we deliberately split this into two steps generate and apply so that it was easy for users to customize.